cpp/source/deal.II/geometry.h

```diff
@@ -72,6 +81,13 @@ class Geometry
     return _materials;
   }
 
+  void set_materials(
+      std::shared_ptr<std::unordered_map<
+          std::string, std::vector<dealii::types::material_id>>> materials)
+  {
+    _materials = materials;
+  }
+
```

Comment:
what's the reason for having separates ``set_anode_boundary_id(...)`` and ``set_cathode_boundary_id`` rather than a
```
void set_materials(
    std::shared_ptr<std::unordered_map<
        std::string, std::vector<dealii::types::boundary_id>>> boundaries)
{ ... }
```